docs/reference/mapping/types/text.asciidoc

```diff
@@ -221,9 +221,11 @@ NOTE: Reordering text fields can have an effect on <<query-dsl-match-query-phras
 aggregations, sorting, or scripting. If you try to sort, aggregate, or access
 values from a script on a `text` field, you will see this exception:
 
+```
 Fielddata is disabled on text fields by default. Set `fielddata=true` on
 `your_field_name` in order to load fielddata in memory by uninverting the
 inverted index. Note that this can however use significant memory.
+```
```

Comment:
As @cbuescher suggests, this text could change over time. I would instead change this to read:

>If you try to sort, aggregate, or access values from a script on a `text` field, you'll see an exception 

> indicating that field data is disabled by default on text fields. To load field data in memory, set `fielddata=true` on field name.
> 
> NOTE: Loading field data in memory can consume significant memory.